README.md

```diff
@@ -11,7 +11,7 @@ Tightly integrated with GitLab, GitHub, and Bitbucket, Gitpod automatically and
 
 ## Features
 
- - Dev environment as code
+ - Dev environments as code
```

Comment:
**thought**: Maybe it's worth taking a fresh look at this list and update accordingly. Listing distinct features could also help.